lighthouse-core/scripts/gcp-collection/gcp-run.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -euxo pipefail
```

Comment:
is the pipefail just for the find / tar command?

Reply:
`set -euxo pipefail` at this point is a habit of putting at the top of every bash script I write :)

in this file yeah AFAICT that's the only command affected by -o pipefail